profiles/package.mask

```diff
@@ -33,6 +33,14 @@
 
 #--- END OF EXAMPLES ---
 
+# Volkmar W. Pogatzki <gentoo@pogatzki.net> (2024-01-15)
+# Unused Java libraries. Removal on 2024-02-15
+dev-java/glassfish-persistence
+dev-java/osgi-core-api
+dev-java/osgi-enterprise-api
+dev-java/osgi-compendium
+dev-java/osgi-foundation
```

Comment:
are you sure we won't need the osgi stuff? that library is pretty common afaik.

Reply:
> are you sure we won't need the osgi stuff? that library is pretty common afaik.

they are all part of either `osgi-core` or `osgi-cmpn`. no need to keep them